lib/Map/MapboxVectorCanvasTileLayer.ts

```diff
@@ -14,11 +18,80 @@ export default class MapboxVectorCanvasTileLayer extends L.GridLayer {
   readonly _zSubtract: number = 0;
   readonly _previousCredits: unknown[] = [];
 
+  @observable splitDirection = ImagerySplitDirection.NONE;
+  @observable splitPosition: number = 0.5;
+
   constructor(
+    private leaflet: Leaflet,
     readonly imageryProvider: MapboxVectorTileImageryProvider,
     options: L.GridLayerOptions
   ) {
     super(Object.assign(options, { async: true, tileSize: 256 }));
+
+    const disposeSplitterReaction = this._reactToSplitterChange();
+    this.on("remove", disposeSplitterReaction);
+  }
+
+  _reactToSplitterChange() {
+    return autorun(() => {
+      const container = this.getContainer();
+      if (container === null) {
+        return;
+      }
+
+      let display = null;
+      if (useClipUpdateWorkaround) {
+        display = container.style.display;
+        container.style.display = "none";
+        container.getBoundingClientRect();
+      }
+
+      if (this.splitDirection === ImagerySplitDirection.LEFT) {
+        const { left: clipLeft } = this._clipsForSplitter;
+        container.style.clip = clipLeft;
+      } else if (this.splitDirection === ImagerySplitDirection.RIGHT) {
+        const { right: clipRight } = this._clipsForSplitter;
+        container.style.clip = clipRight;
+      } else {
+        container.style.clip = "auto";
+      }
+
+      if (useClipUpdateWorkaround) {
+        container.style.display = display!;
+      }
+    });
+  }
+
+  @computed
+  get _clipsForSplitter() {
+    let clipLeft = "";
+    let clipRight = "";
+    let clipPositionWithinMap;
+    let clipX;
+
+    if (this.leaflet.size && this.leaflet.nw && this.leaflet.se) {
+      clipPositionWithinMap = this.leaflet.size.x * this.splitPosition;
+      clipX = Math.round(this.leaflet.nw.x + clipPositionWithinMap);
+      clipLeft =
+        "rect(" +
+        [this.leaflet.nw.y, clipX, this.leaflet.se.y, this.leaflet.nw.x].join(
+          "px,"
+        ) +
+        "px)";
+      clipRight =
+        "rect(" +
+        [this.leaflet.nw.y, this.leaflet.se.x, this.leaflet.se.y, clipX].join(
+          "px,"
+        ) +
+        "px)";
+    }
+
+    return {
+      left: clipLeft,
+      right: clipRight,
+      clipPositionWithinMap: clipPositionWithinMap,
+      clipX: clipX
+    };
```

Comment:
This is all practically the same as in `CesiumTileLayer` - https://github.com/TerriaJS/terriajs/blob/a76974baabcb8cc4a12bae09ab32d497c3d3d89c/lib/Map/CesiumTileLayer.ts